lib/lte_link_control/lte_lc.c

```diff
@@ -125,6 +125,20 @@ static const char nw_mode_preferred[] = "AT%XSYSTEMMODE=1,0,1,0";
 static const char nw_mode_fallback[] = "AT%XSYSTEMMODE=0,1,1,0";
 #endif
 
+#if defined(CONFIG_LWM2M_CARRIER) && !defined(CONFIG_GPS_USE_SIM)
+#if defined(CONFIG_BOARD_NRF9160_PCA20035NS)
+	const char *const lwm2m_ant_cfg[] = {
+			"AT%XMAGPIO=1,1,1,7,1,746,803,2,698,748,"
+			"2,1710,2200,3,824,894,4,880,960,5,791,849,"
+			"7,1574,1577",
+			"AT%XMODEMTRACE=0"};
```

Comment:
I think this one should be removed (also from `board_nonsecure.c`, but possibly out of scope here). We do support tracing for Thingy with the updated connectivity bridge.

Reply:
removed modem trace command.
perhaps another PR for changes to `board_nonsecure.c`